web-ui/src/pages/FeedbackSubmitPage.jsx

```diff
@@ -1,14 +1,19 @@
 import React from "react";
+import { useState } from 'react'
 import "./FeedbackRequestPage.css";
 import FeedbackSubmissionTips from "../components/feedback_submission_tips/FeedbackSubmissionTips";
-
-
+import FeedbackSubmitForm from "../components/feedback_submit_form/FeedbackSubmitForm";
 
 const FeedbackSubmitPage = () => {
 
+  const [showTips, setShowTips] = useState(true);
+
   return (
     <div className="feedback-submit-page">
-      <FeedbackSubmissionTips/>
+      {showTips ?
+        <FeedbackSubmissionTips onNextClick={() => setShowTips(false)} /> :
+        <FeedbackSubmitForm requesteeName={"JohnDoe"}/>
+      }
     </div>
```

Comment:
Wouldn't we want to dynamically pass in the requestee name here, similar to how we pass it in for the request?

Reply:
Yes, that is the eventual plan, but we've made a separate issue for that (feature #1178). This feature is more concerned with the view rather than the functionality.

Reply:
Okay, perfect.